inst/include/Rcpp/sugar/functions/complex.h

```diff
@@ -68,6 +68,7 @@ inline double complex__Re( Rcomplex x){ return x.r ; }
 		y.i = -x.i ;
 		return y ;
 	}
+    inline double complex__Arg( Rcomplex x ){ return ::atan2(x.i, x.r); }
```

Comment:
The vertical alignment seems to a little out of whack but we can fix that later.